src/utils/geometry.ts

```diff
@@ -143,10 +157,10 @@ export interface ArcGeometry {
   };
 }
 
-export function isPointGeometry(ig: IndexedGeometry): ig is PointGeometry {
+export function isPointGeometry(ig: IndexedGeometry): ig is PointGeometry & { panel: Dimensions } {
```

Comment:
Shouldn't this be...

```suggestion
export function isPointGeometry(ig: IndexedGeometry): ig is PerPanel<PointGeometry> {
```

Reply:
done in https://github.com/elastic/elastic-charts/pull/793/commits/5bfda72ae99d371ed617b3ccd877ae99252d714e